src/components/orders/OrderNotFound/index.tsx

```diff
@@ -118,6 +127,11 @@ export const OrderAddressNotFound: React.FC = (): JSX.Element => {
           </SearchContent>
         </SearchSection>
       </Content>
+      {showLinkData && (
+        <LinkData>
+          This is not a CowProtocol transaction. See it in the <BlockExplorerLink {...(data as never)} />
```

Comment:
```suggestion
          This is not a CowProtocol transaction. See it on <BlockExplorerLink {...(data as never)} />
```

Reply:
Also, why casting `data` as `never`?

Reply:
@alfetopito I used  `never` in order to keep data as  a generic type in case it is used as a property for sharing extra information to the component. I used `any` at first but eslint recommend me to cast it to `never`